spec/order_spec.rb

```diff
@@ -0,0 +1,41 @@
+require './lib/order'
+require "./lib/menu"
+require "./lib/user"
+
+describe Order do
+
+  let(:item) { double :order }
```

Comment:
Great to see some doubles in your tests - I really should've spent time implementing these as well.